<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JsonToYaml</class>
 <widget class="QWidget" name="JsonToYaml">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>824</width>
        <height>758</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_12">
       <item row="0" column="0">
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QGridLayout" name="gridLayout_11">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="titleLabel">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>JSON-YAML数据类型互转工具</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="setGroupBox">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>150</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>150</height>
             </size>
            </property>
            <property name="title">
             <string>配置</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <widget class="QFrame" name="frame">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>70</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="gridLayout_2">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label">
                    <property name="minimumSize">
                     <size>
                      <width>30</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>30</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="pixmap">
                     <pixmap resource="devtoys.qrc">:/DevToys/icon/转换.png</pixmap>
                    </property>
                    <property name="scaledContents">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLabel" name="label_2">
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>70</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>转换
选择您想要使用的转换模式</string>
                    </property>
                    <property name="wordWrap">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="2">
                   <widget class="QComboBox" name="ConvertComboBox">
                    <property name="maximumSize">
                     <size>
                      <width>150</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <item>
                     <property name="text">
                      <string>JSON转换到YAML</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>YAML转换到JSON</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QFrame" name="frame_2">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>70</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="gridLayout_5">
                  <item row="0" column="1">
                   <widget class="QLabel" name="label_4">
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>70</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>缩进</string>
                    </property>
                    <property name="wordWrap">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_3">
                    <property name="minimumSize">
                     <size>
                      <width>30</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>30</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="pixmap">
                     <pixmap resource="devtoys.qrc">:/DevToys/icon/缩进.png</pixmap>
                    </property>
                    <property name="scaledContents">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="2">
                   <widget class="QComboBox" name="indentComboBox">
                    <property name="maximumSize">
                     <size>
                      <width>150</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <item>
                     <property name="text">
                      <string>四个空格</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>两个空格</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QSplitter" name="splitter">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QWidget" name="widget_3" native="true">
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <widget class="QWidget" name="widget" native="true">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
                <layout class="QGridLayout" name="gridLayout_7">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item row="0" column="0">
                  <layout class="QHBoxLayout" name="horizontalLayout">
                   <property name="spacing">
                    <number>10</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="label_5">
                     <property name="text">
                      <string>输入</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pasteButton">
                     <property name="minimumSize">
                      <size>
                       <width>50</width>
                       <height>25</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>50</width>
                       <height>25</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>粘贴</string>
                     </property>
                     <property name="icon">
                      <iconset resource="devtoys.qrc">
                       <normaloff>:/DevToys/icon/粘贴.png</normaloff>:/DevToys/icon/粘贴.png</iconset>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="OpenFileButton">
                     <property name="minimumSize">
                      <size>
                       <width>25</width>
                       <height>25</height>
                      </size>
                     </property>
                     <property name="toolTip">
                      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;打开文件&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset resource="devtoys.qrc">
                       <normaloff>:/DevToys/icon/打开文件.png</normaloff>:/DevToys/icon/打开文件.png</iconset>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="ClearButton">
                     <property name="minimumSize">
                      <size>
                       <width>25</width>
                       <height>25</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>25</width>
                       <height>25</height>
                      </size>
                     </property>
                     <property name="toolTip">
                      <string>清空</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset resource="devtoys.qrc">
                       <normaloff>:/DevToys/icon/清空.png</normaloff>:/DevToys/icon/清空.png</iconset>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="SciScintilla" name="importWidget" native="true"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="widget_4" native="true">
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="0" column="0">
               <widget class="QWidget" name="widget_2" native="true">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
                <layout class="QGridLayout" name="gridLayout_10">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item row="0" column="0">
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                   <property name="spacing">
                    <number>10</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="label_6">
                     <property name="text">
                      <string>输出</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_2">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QPushButton" name="expandButton">
                     <property name="minimumSize">
                      <size>
                       <width>25</width>
                       <height>25</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>25</width>
                       <height>25</height>
                      </size>
                     </property>
                     <property name="toolTip">
                      <string>展开/收缩</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset resource="devtoys.qrc">
                       <normaloff>:/DevToys/icon/展开.png</normaloff>:/DevToys/icon/展开.png</iconset>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="copyButton">
                     <property name="minimumSize">
                      <size>
                       <width>50</width>
                       <height>25</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>50</width>
                       <height>25</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>复制</string>
                     </property>
                     <property name="icon">
                      <iconset resource="devtoys.qrc">
                       <normaloff>:/DevToys/icon/复制.png</normaloff>:/DevToys/icon/复制.png</iconset>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="SciScintilla" name="exportsWidget" native="true"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SciScintilla</class>
   <extends>QWidget</extends>
   <header>sciscintilla.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="devtoys.qrc"/>
 </resources>
 <connections/>
</ui>
